linera-storage/src/lib.rs

```diff
@@ -429,8 +429,18 @@ where
         }
     }
 
-    async fn get_blob(&self, blob_id: BlobId) -> Result<Blob, ViewError> {
-        self.storage.read_blob(blob_id).await
+    async fn get_blobs(&self, blob_ids: BlobId) -> Result<Blob, ViewError> {
+        let maybe_blobs = self.storage.read_blobs(blob_ids).await;
```

Comment:
How does it work - `blob_ids` is still a single `Blobid` type.

Reply:
(Just wasn't done yet. Fixed now.)